plugins/woocommerce-blocks/assets/js/blocks/product-collection/frontend.tsx

```diff
@@ -81,53 +93,32 @@ function scrollToFirstProductIfNotVisible( wcNavigationId?: string ) {
 }
 
 const productCollectionStore = {
-	state: {
-		get startAnimation() {
-			return (
-				getContext< ProductCollectionStoreContext >().animation ===
-				'start'
-			);
-		},
-		get finishAnimation() {
-			return (
-				getContext< ProductCollectionStoreContext >().animation ===
-				'finish'
-			);
-		},
-	},
 	actions: {
 		*navigate( event: MouseEvent ) {
-			const ctx = getContext< ProductCollectionStoreContext >();
 			const { ref } = getElement();
-			const wcNavigationId = (
-				ref?.closest( '[data-wp-router-region]' ) as HTMLDivElement
-			 )?.dataset?.wcNavigationId;
-			const isDisabled = (
-				ref?.closest( '[data-wp-router-region]' ) as HTMLDivElement
-			 )?.dataset.wcNavigationDisabled;
-
-			if ( isDisabled ) {
+
+			if ( ! ref ) {
+				return;
+			}
+
+			const ctx = getContext< ProductCollectionStoreContext >();
```

Comment:
Nitpick: We could move this into the `if()` condition so we don't execute unnecessarily

Reply:
https://github.com/woocommerce/woocommerce/pull/55330/commits/e3cd2611fe0157a48ca150821024bd8145b4778d